editor/xml_feature.hpp

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "coding/multilang_utf8_string.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "std/ctime.hpp"
```

Comment:
I do need ctime for ::time, point2d for PointD, and iostream for ostream which is declared in std/iostream.hpp. And I don't really need map. Thanks for that.